imagersite/imager_images/templates/library.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+{% load thumbnail %}
+{% block content %}
+
+{{ STATIC_URL }}
+
+<div class='imager-library'>
+
+<h2>{{ user }}</h2>
+
+<h3>All Albums</h3>
+
+{% for album in user.albums.all %}
+<div class='all_photos'>
+<h3>{{ album.title }}</h3>
+{% if album.cover %}
+    {% thumbnail album.cover.file "180x180" as im %}
+    <a href="{% url 'album_detail' album.id %}"><img src='{{ MEDIA_ROOT }}{{ im.url }}'/></a>
+    {% endthumbnail %}
+
+{% else %}
+<a href="{% url 'album_detail' album.id %}"><img src='/static/thumbnail.jpg' height='180' width='180'></a>
+{% endif %}
+{% cycle "" "" "<br/>" %}
```

Comment:
Ahhh, yes.  Here the tag is inside the for loop.  Makes much more sense.  I think the other usage above must just be a mistake.